ipfsconn/ipfshttp/ipfshttp.go

```diff
@@ -92,6 +92,22 @@ type ipfsAddResp struct {
 	Bytes uint64
 }
 
+type ipfsSwarmPeersResp struct {
+	Peers []ipfsPeer
+}
+
+type ipfsPeer struct {
+	Addr    string
+	Peer    string
+	Latency string
+	Muxer   string
+	Streams []ipfsStream
```

Comment:
If you're not interested in the streams, you can just not include the key in your struct and the json parser will ignore it altogether.

Reply:
(i remember posting here this too): Might work with just 

```
type ipfsSwarmPeersResp struct {
  Peers struct{
     Peer string
  }
}
}